pallets/support/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "pallets-support"
```

Comment:
Any particular reason to name it like this? Ie is it some substrate convention? I was thinking of anything `migration` in the name

Reply:
I'm expecting that more things may appear here, so in general: support (like frame-support)